<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabScanScheduling</class>
 <widget class="QWidget" name="TabScanScheduling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Prüfungsintervall</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Wie oft soll das Benutzerverzeichnis überprüft werden:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Verzeichnisse</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listWidget_Directories">
        <property name="toolTip">
         <string>Zu überprüfende Verzeichnisse.</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_OpenDir">
          <property name="toolTip">
           <string>Verzeichnisse auswählen, deren Inhlate von ClamAV überprüft werden sollen.</string>
          </property>
          <property name="text">
           <string>Verzeichnisse hinzufügen</string>
          </property>
          <property name="icon">
           <iconset theme="document-open-folder"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ClearList">
          <property name="toolTip">
           <string>Ausgewählte Verzeichnisse entfernen.</string>
          </property>
          <property name="text">
           <string>Verzeichnisse Entfernen</string>
          </property>
          <property name="icon">
           <iconset theme="edit-clear-list"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Automatisches aktuallisieren der Virendatenbank</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>Wie häufig soll nach einer aktuellen Virendatenbank gesucht werden?</string>
          </property>
          <property name="text">
           <string>Wie oft am Tag:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_FreshClamUpdate">
          <property name="toolTip">
           <string>Wie häufig soll nach einer aktuellen Virendatenbank gesucht werden?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox_FreshClam">
          <property name="toolTip">
           <string>Startet das Programm freshclam im Hintergrund und überprüft in regelmäßigen Abständen die Aktuallität der Virendefinitionen. </string>
          </property>
          <property name="text">
           <string>freshclam im Hintergrund starten.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
